spec/models/update_services_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe UpdateServices do
+  context ".update_services" do
+    it "calls the appropriate methods to ping and process a ping" do
+      allow(UpdatedServicesPinger).to receive(:ping)
+        .and_return(:fake_object)
+      allow(UpdatedServicesProcessor).to receive(:process)
+        .with(:fake_object)
+        .and_return(:fake_result)
```

Comment:
when are you are testing that other methods get called like this consider adding a specific number of times you expect they will be called. Otherwise, I'm not sure the tests would fail if they aren't called. Even so, the explicit testing could help if a method ends up in a loop or something. So that would be like:
```ruby
allow(UpdatedServicesPinger).to receive(:ping).once.and_return(:fake_object)
allow(UpdatedServicesProcessor).to receive(:process).once.with(:fake_object).and_return(:fake_result)
```